gno.land/cmd/gnoland/testdata/addpkg.txtar

```diff
@@ -0,0 +1,26 @@
+# test for add package
+
+## start a new node
+gnoland start
+
+## add bar.gno package located in $WORK directory as gno.land/r/foobar/bar
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/foobar/bar -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+## execute Render
+gnokey maketx call -pkgpath gno.land/r/foobar/bar -func Render -gas-fee 1000000ugnot -gas-wanted 2000000 -args '' -broadcast -chainid=tendermint_test test1
```

Comment:
ideally we should make all the parameters optional with a default:
- `-chainid` should always be tendermint_test, we don't care about testing other values in the integration tests, it belongs to unit tests instead. I suggest always setting this value, and eventually log a warning if the value is different.
- `-broadcast` is mandatory, because we can't chain the airgap-method with txtar.
- `-gas-wanted` and `-gas-fee` should stay like this IMO (no default value).

By doing this, we can copy-paste commands from the terminal.

Reply:
thanks for the precision ! 
I totally agree on doing that,  but can I do this in a next iteration? for now I only add default arguments on the root level of gnokey, which contain only: 
```
  -config ...                                           
  -home  /Users/XXX/Library/Application Support/gno  
  -insecure-password-stdin=false                        
  -quiet=false                                          
  -remote 127.0.0.1:26657                              
```

args like `chain-id` are ont a second level, which require checking the kind of subcommand, ignore potential user args and finally properly inject it at the right place (not that difficult to do, but a bit fastidious).
If it's really mandatory for this PR to be merge I can take some time to add this.